### PR TITLE
fix: fix EC issues in release 2.7

### PR DESCRIPTION
**Pull request by ernesgonzalez33**

### CODE REVIEW

**Comment by ernesgonzalez33** [comment]
@mnecas @yaacov PTAL and merge if everything OK

File: build/virt-v2v/Containerfile [hunk 1]
```diff
@@ -1,40 +1,51 @@
...
 WORKDIR /app
 COPY --chown=1001:0 ./ ./
 ENV GOFLAGS "-mod=vendor -tags=strictfipsruntime"
 ENV GOEXPERIMENT strictfipsruntime
 
-RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o virt-v2v-monitor ./cmd/virt-v2v-monitor/virt-v2v-monitor.go
-RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o image-converter ./cmd/image-converter/image-converter.go
-RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o virt-v2v-wrapper ./virt-v2v/cmd/entrypoint.go
+RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o virt-v2v-monitor github.com/konveyor/forklift-controller/cmd/virt-v2v-monitor
+RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o image-converter github.com/konveyor/forklift-controller/cmd/image-converter
```
Comment (yaacov) [inline build/virt-v2v/Containerfile:26]:
```suggestion
RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o virt-v2v-monitor github.com/konveyor/forklift-controller/cmd/virt-v2v-monitor/virt-v2v-monitor.go
RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o image-converter github.com/konveyor/forklift-controller/cmd/image-converter/image-converter.go
```

File: build/virt-v2v/Containerfile-upstream
Comment (yaacov) [inline build/virt-v2v/Containerfile-upstream:1]: can we have one container file that will work both upstream and downstream ?

Comment (ernesgonzalez33) [reply build/virt-v2v/Containerfile-upstream:1]: Yes, by using arguments. But we are right now focusing only on upstream. I kept that one to don't lose the code

**Comment by ernesgonzalez33** [comment]
@yaacov your requested change is failing the build:

```
[2/3] STEP 6/8: RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o virt-v2v-monitor github.com/konveyor/forklift-controller/cmd/virt-v2v-monitor/virt-v2v-monitor.go
cannot find module providing package github.com/konveyor/forklift-controller/cmd/virt-v2v-monitor/virt-v2v-monitor.go: import lookup disabled by -mod=vendor
subprocess exited with status 1
subprocess exited with status 1
Error: building at STEP "RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o virt-v2v-monitor github.com/konveyor/forklift-controller/cmd/virt-v2v-monitor/virt-v2v-monitor.go": exit status 1
```

File: build/virt-v2v/Containerfile [hunk 2]
```diff
@@ -1,40 +1,51 @@
...
+FROM registry.access.redhat.com/ubi9/go-toolset:1.21.13-2.1727893526 AS builder
 WORKDIR /app
 COPY --chown=1001:0 ./ ./
 ENV GOFLAGS "-mod=vendor -tags=strictfipsruntime"
 ENV GOEXPERIMENT strictfipsruntime
 
-RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o virt-v2v-monitor ./cmd/virt-v2v-monitor/virt-v2v-monitor.go
-RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o image-converter ./cmd/image-converter/image-converter.go
-RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o virt-v2v-wrapper ./virt-v2v/cmd/entrypoint.go
+RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o virt-v2v-monitor github.com/konveyor/forklift-controller/cmd/virt-v2v-monitor/virt-v2v-monitor.go
```
Comment (mnecas) [inline build/virt-v2v/Containerfile:25]: @ernesgonzalez33 please remove the `virt-v2v-monitor.go`

File: build/virt-v2v/Containerfile [hunk 3]
```diff
@@ -1,40 +1,51 @@
...
+RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o image-converter github.com/konveyor/forklift-controller/cmd/image-converter/image-converter.go
```
Comment (mnecas) [inline build/virt-v2v/Containerfile:26]: @ernesgonzalez33 please remove the `image-converter.go`

**Pull request merged**